runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ActiveWorkState.java

```diff
@@ -202,19 +193,19 @@ synchronized void failWorkForKey(Multimap<Long, WorkId> failedWork) {
    *     cause a {@link java.util.ConcurrentModificationException} as it is not a thread-safe data
    *     structure.
    */
-  synchronized ImmutableListMultimap<ShardedKey, RefreshableWork> getReadOnlyActiveWork() {
+  synchronized ImmutableListMultimap<Long, RefreshableWork> getReadOnlyActiveWork() {
```

Comment:
It appears that this method is just used by ComputationState method that is not called, can you remove them both?

Reply:
done.